changelogs/current.yaml

```diff
@@ -62,6 +62,9 @@ new_features:
 - area: gzip
   change: |
     added support for :ref:`max_inflate_ratio<envoy_v3_api_msg_extensions.compression.gzip.decompressor.v3.Gzip>`.
+- area: http
```

Comment:
Based on discussion about l4 vs l7 logging, I think the `area` is incorrect. Maybe it should be `access_log`?

Reply:
Done, thanks.